k8sutils/redis-cluster.go

```diff
@@ -184,3 +185,21 @@ func getRedisLabels(name, setupType, role string) map[string]string {
 		"role":             role,
 	}
 }
+
+// generateSidecars generates Redis sidecar informations
+func generateClusterSidecars(cr *redisv1beta1.RedisCluster /*cr *redisv1beta1.Redis*/) []sidecarParameters {
+	sidecars := []sidecarParameters{}
+
+	if cr.Spec.Sidecars != nil {
```

Comment:
```suggestion
	if cr.Spec.Sidecars == nil {
		return sidecars
	}
```

nit: You could reverse this condition to prevent indenting the for-loop.

Reply:
To build on this: the idiomatic way to handle error in Go is to fail fast and exit the function. Makes for easier to read code, as the further along you are in the function the further you are in the happy path.